vitrina/datasets/factories.py

```diff
@@ -15,6 +14,15 @@ class Meta:
     will_be_financed = False
 
 
+class DatasetMemberFactory(DjangoModelFactory):
+    class Meta:
+        model = DatasetMember
+        django_get_or_create = ('role',)
+
+    role = factory.Faker('word')
```

Comment:
```suggestion
    role = DatasetMember.CREATOR
```

One of available choice must be used.